rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1134,6 +1135,9 @@ private void processPegoutsInBatch(
             // TODO: Update to call addPegoutTxToReleaseTransactionSet with the RskHash that calls the updateCollections
             releaseTransactionSet.add(generatedTransaction, rskExecutionBlock.getNumber());
 
+            // Remove All requests on the queue after successfully batching pegouts
+            releaseRequestQueue.removeEntries(pegoutEntries);
```

Comment:
This may not be correct if we implement the _partial_ batching (i.e. creating a batch up to the limit of tx size) as there could be pegout requests waiting to be processed. You may want to leave a TODO here...

Reply:
Added a TODO Here. Thanks Jose